### PR TITLE
Maven cleanup

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -139,10 +212,6 @@
 			<id>dmulloy2-repo</id>
 			<url>http://repo.dmulloy2.net/content/groups/public</url>
 		</repository>
-		<repository>
-			<id>jitpack.io</id>
-			<url>https://jitpack.io</url>
-		</repository>
```
Comment (RoboTricker) [inline pom.xml:145]: Why did you remove this and some more repos? Because they are hosted on the codemc repo?

Comment (sgdc3) [reply pom.xml:145]: I didn't find any dependency that wasn't available in the other repos
